docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>I understand why we are showing this but, in its current form, I don't think very few people will actually try to look for the character. I don't mean to add more work for you but would it make any sense to place these characters in context within the time series above? Or we could omit it?  Not trying to dictate here. Just thinking out loud and open to discussion!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Yeah I agree. I think putting them into the visualisations would require me to also find the traces for those other characters, which is too much work and will probably crowd the plot. How about highlighting that matching character in the print output, like</p><p> </p><p><code>from IPython.display import Markdown, display</code></p><p><code>display(Markdown('foo **bar** baa'))</code></p><p><code>display(Markdown('abc dfg <strong>**bar**</strong>'))</code></p><p> </p><p>This way, people won't have such a hard time finding the match.</p><p> </p><p>Or, I could use the sentences as y-tick labels (with the match highlighted, if that's possible for tick labels) in the plot above... Maybe that's even better because you don't have to make the active decision to skip over the print output.</p>

Reply:
<p>I had not thought about the y-tick labels and I kinda like that idea. Otherwise, <code>display</code>  is fine too. I trust your judgement!</p>